lib/shared/bucketing-assembly-script/assembly/types/configBody.ts

```diff
@@ -13,30 +13,44 @@ import { Feature } from './feature'
 import { Audience } from './target'
 
 export class PublicProject extends JSON.Value {
-    readonly _id: string
-    readonly key: string
-    readonly a0_organization: string
-    readonly settings: JSON.Obj
+    readonly _id: string;
+    readonly key: string;
+    readonly a0_organization: string;
+    disablePassthroughRollouts: bool; // Now part of PublicProject directly
 
     constructor(project: JSON.Obj) {
-        super()
-        this._id = getStringFromJSON(project, '_id')
-        this.key = getStringFromJSON(project, 'key')
-        this.a0_organization = getStringFromJSON(project, 'a0_organization')
-        this.settings = getJSONObjFromJSON(project, 'settings')
+        super();
+        this._id = getStringFromJSON(project, '_id');
+        this.key = getStringFromJSON(project, 'key');
+        this.a0_organization = getStringFromJSON(project, 'a0_organization');
+
+        // Parse settings directly within this class
+        let settings = getJSONObjFromJSON(project, 'settings');
```

Comment:
Maybe `ProjectSettings` should be its own Class and parsed the same way the other sub-objects are?